internal/api/v1beta1/app_types.go

```diff
@@ -212,6 +212,9 @@ type AppSpec struct {
 
 	// Annotations is a list of annotations that will be applied to processes
 	Annotations []MetadataItem `json:"annotations,omitempty"`
+
+	// Secret is the secret TLS certifificate to associate with the App
+	SecretName string `json:"secretName,omitempty"`
```

Comment:
is it possible to use an array of secrets?

Reply:
You bet. I made the change and tested out multiple secrets.